extension.js

```diff
@@ -135,8 +135,14 @@ const ScrollBox = new Lang.Class({
         let dueDateAbbreviation = task.DueDateAbbreviation;
 
         let description = (dueDateAbbreviation ? dueDateAbbreviation + "  " : "") + task.Description
+        if (task.started){
+            description = "⭐ " + description;
```

Comment:
This was added just for debug purposes. But I kinda like it at the end, so I decided to leave it. Remove if you want to.